modules/claims_api/lib/claims_api/v2/params_validation/power_of_attorney/poa_submission_validator.rb

```diff
@@ -18,6 +18,8 @@ def validate_poa_code(record)
           end
 
           def validate_signatures(record)
+            record.data[:signatures] = { veteran: 'sign_here', representative: 'sign_here' }
+            # TODO: setup signature method in controller
```

Comment:
Is this a like _another ticket_ to-do? Not sure this needs to be left here, but perhaps there is some context I don't know.